TombEngine/Game/camera.cpp

```diff
@@ -58,8 +58,15 @@ GameVector LookCamPosition;
 GameVector LookCamTarget;
 Vector3i CamOldPos;
 CAMERA_INFO Camera;
+OBJ_CAMERA_INFO ItemCamera;
 GameVector ForcedFixedCamera;
 int UseForcedFixedCamera;
+ItemInfo* camSlotId;
```

Comment:
I think that all new fields here also should go to OBJ_CAMERA_INFO struct. It will clearly say to other developers that these fileds are only covering a camera which is bound to moveable mesh.

Reply:
I removed all unnecessary variables and put the needed ones into the struct.